services/sysinfo/server/sysinfo_linux.go

```diff
@@ -20,21 +20,58 @@ Copyright (c) 2023 Snowflake Inc. All rights reserved.
 package server
 
 import (
+	"bufio"
+	"context"
+	"encoding/json"
 	"fmt"
+	"strconv"
 	"time"
 
 	pb "github.com/Snowflake-Labs/sansshell/services/sysinfo"
+	"github.com/Snowflake-Labs/sansshell/services/util"
+	"github.com/Snowflake-Labs/sansshell/telemetry/metrics"
 	"github.com/euank/go-kmsg-parser/v2/kmsgparser"
+	"go.opentelemetry.io/otel/attribute"
 	"golang.org/x/sys/unix"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
 // for testing
-var getKmsgParser = func() (kmsgparser.Parser, error) {
-	return kmsgparser.NewParser()
-}
+var (
+	journalctlBin                   = "/usr/bin/journalctl"
+	expectedTimeFormatForJournalctl = "2006-01-02 15:04:05"
```

Comment:
services/sysinfo/client/client.go:247 - you already have this - can you reuse this?

Reply:
Oh, almost forget, let me reuse the variable.